src/Components/Components/src/EventCallbackFactoryBinderExtensions.cs

```diff
@@ -0,0 +1,403 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+
+namespace Microsoft.AspNetCore.Components
+{
+    /// <summary>
+    /// Contains extension methods for two-way binding using <see cref="EventCallback"/>. For internal use only.
+    /// </summary>
+    public static class EventCallbackFactoryBinderExtensions
+    {
+        /// <summary>
+        /// For internal use only.
+        /// </summary>
+        /// <param name="factory"></param>
+        /// <param name="receiver"></param>
+        /// <param name="setter"></param>
+        /// <param name="existingValue"></param>
+        /// <returns></returns>
+        public static EventCallback<UIChangeEventArgs> CreateBinder(
+            this EventCallbackFactory factory,
+            object receiver,
+            Action<string> setter,
+            string existingValue)
+        {
+            Func<object, string> converter = (obj) => (string)obj;
```

Comment:
Does this method get called by generated code at each `bind` call site?

If so, instead of instantiating a new `converter` on each call, can we cache the `Func<object, *>` instances on static fields for reuse?

Reply:
Here and in the other `CreateBinder` methods, that is.

Reply:
If you'd feel better I can move these to static fields so that it's obvious. 

The compiler already does this optimization where possible. There are different preferences for how deal with this problem when you want to be *sure* that the lambda is static, and they all have drawbacks, other than static local functions which requires C# 8.